pom.xml

```diff
@@ -88,9 +88,8 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-databind</artifactId>
-			<version>${jackson.version}</version>
-		</dependency>
- 
+			<version>2.13.4.2</version>
```

Comment:
The version variable is used multiple times. Seems that it should be updated elsewhere or this and other jackson dependencies may be incompatible with each other.